lib/interfaces/jwt-module-options.interface.ts

```diff
@@ -12,8 +12,8 @@ export enum JwtSecretRequestType {
 export interface JwtModuleOptions {
   global?: boolean;
   signOptions?: jwt.SignOptions;
-  secret?: string | Buffer;
-  publicKey?: string | Buffer;
+  secret?: jwt.Secret;
```

Comment:
are you sure that we can still assign `string` or `Buffer` to these `jwt.Secret` fields? I didn't check the types